reconcile/utils/terraform_client.py

```diff
@@ -400,7 +402,7 @@ def format_output(self, output, type):
             return data[list(data.keys())[0]]
         return data
 
-    def construct_oc_resource(self, name, data, account):
+    def construct_oc_resource(self, name, data, annotations, account):
```

Comment:
Why aren't we putting this annotations field at the end of the argument list? It's pretty easy to get confused and update callers the wrong way otherwise...

Also, in all the other updates you've chosen to return or pass annotations at the end of the relevant list.

Reply:
Happy to move it to the end.